Task_HelloWorld/index.html

```diff
@@ -1,15 +1,94 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>Hello World Problem</title>
-    <link rel="stylesheet" href="index.css" />
-  </head>
-
-  <body style="margin: 0; padding: 0">
-    <div top="0">
-      <a href="../index.html">Go Back</a>
-    </div>
-
-    <!-- Start coding below FOR HELLO WORLD TASK;  DO NOT TOUCH the code above -->
-  </body>
-</html>
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Hello World Problem</title>
+    <link rel="stylesheet" href="index.css" />
+  </head>
+
+  <body style="margin: 0; padding: 0">
+    <div top="0">
+      <a href="../index.html">Go Back</a>
+    </div>
+
+    <!-- Start coding below FOR HELLO WORLD TASK;  DO NOT TOUCH the code above -->
+     <h1>Hello World</h1>
+
+    <p><b> This is bold tag & element</b><br>
+          <i> This is itailic tag & element</i><br>
+          <u>This is Underline tag & element </u><br>
+ </p>
+
+ <pre>
+     I
+     am
+    developer
+ </pre>
+ <br>
+
+ <p>H <sub>2</sub>O </p><br>
+ <p>A <sup>4</sup></p><br>
+
+ <hr>
+
+
+ <p>making portfolio....</p>
+
+ <header>
+        <h1>Priyanka  Kumari</h1>
+         <img src="https://i.pinimg.com/736x/de/6f/21/de6f21a62c5414ec1f65b41afc173539.jpg" alt="priyanka-photo" width="150" >
+        <p>Web Developer & Software Developer</p>
+    </header>
+
+    <section>
+        <h2>About me</h2>
+        <p>Welcome to my portfolio.... <br> I am priyanka kumari,I'm web-developer and software developer ,I doing code in html and css as well javaScript and react
+           I'm complited 10th or 12th from bihar bord. I love doing dance and yoga it is I'm doing when I found free from our work I love traveling and advanturing diffrent diffren place
+           .About my family My father is farmer and my mother is house-wife .I've 4 siblings 3 sister and 1 brother ,I'm elder in my siblings
+        </p>
+    </section>
+
+    <section>
+        <h2>About my Skill</h2>
+        <p>I've skill like..</p>
+        <ul>
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>javaScript</li>
+            <li>React</li>
+            <li>SQL</li>
+            <li>Git ...etc</li>
+            
+        </ul>
+    </section>
+   
+    <section>
+        <h2>Education</h2>
+        <p>B.tech-computer science engineering in AGC(amritsar group of college)</p>
+        <p></p>
+    </section>
+
+    <section>
+        <h2>Contect Me</h2>

```

Comment:
The 'footer' element should be placed within a section or article element for proper semantic structure.